### ISSUE TITLE
Challenge suite not passing albeit correct solution

### ISSUE THREAD

**Issue by carvalhais** [freeCodeCamp/freeCodeCamp#38402]
**Describe your problem and how to reproduce it:**
The solution to this challenge seems to have a bug when run inside of Chrome. Even though the solution is correct (as stated by the ‘Hint’), it doens’t pass the suite of tests until you zoom in (with Ctrl+Mouse Wheel) to 150%.
Maybe this is related to the resolution of ones screen?

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-principles/use-a-retina-image-for-higher-resolution-displays

**Tell us about your browser and operating system:**
User Agent is: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36.

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
![image](https://user-images.githubusercontent.com/5495377/77160433-0f1de200-6a86-11ea-98ae-babdcbb1470d.png)

**Comment by hassaanp** [comment]
Hi @carvalhais,

That sounds strange.
I just confirmed that the validation for the problem only checks the values of height and width in the code.

I am guessing there was some other issue, but it definitely can't be due to the resolution of the screen.

**Comment by Sky020** [comment]
@carvalhais , what extensions do you have enabled with access to freeCodeCamp?

**Comment by kevthew** [comment]
im having the same issue here
[](https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-principles/make-an-image-responsive)

I followed along with the video but nothing the first test fails and the last test passes 
heres my
 `<style>
.responsive-img {
  max-width: 100%;
  display: block;
  height: auto;
}

img {
  width: 600px;
}
</style>`

**Comment by Sky020** [comment]
@kevthew , Please note that the code in the video is an example, and not the solution to the challenge. You are not supposed to have `display: block` in the solution.

**Comment by kevthew** [comment]
it worked on chrome with either solution the test is broke on edge

**Comment by carvalhais** [comment]
> @carvalhais , what extensions do you have enabled with access to freeCodeCamp?

Apresentações : version 0_10
Web Store : version 0_2
Documentos : version 0_10
Google Drive : version 14_2
YouTube : version 4_2_8
User-Agent Switcher : version 2_0_0_4
Gmail Off-line : version 1_20
Google Agenda : version 4_5_10
Video Downloader professional : version 1_99_1
Planilhas : version 1_2
Editor do Office : version 134_193_207
Feedback : version 1_0
Documentos Google off-line : version 1_9_1
CryptoTokenExtension : version 0_9_74
Application Launcher for Drive (by Google) : version 3_2
Extensão do Google Keep para o Chrome : version 4_20103_1139_1
Gerenciador de Download Chrono : version 0_10_0
Cloud Print : version 0_1
Chrome PDF Viewer : version 1
Google Network Speech : version 1_0
Google Hangouts : version 1_3_15
Pagamentos da Chrome Web Store : version 1_0_0_5
Gmail : version 8_1
Chrome Media Router : version 8220_313_1_1
Extensão Check-up de senha : version 1_12

**Comment by Sky020** [comment]
@carvalhais That is a lot of extensions. I cannot see any that would interfere, but, just to confirm, I suggest denying access to fCC for all of them.

**Comment by carvalhais** [comment]
So, I have found the culprit, it turns out that's a Windows bug, somehow. The problem manifests whenever you have a custom resolution scaling setting turned on (Start menu → Settings → System → Display → Custom scaling, see attached file, but it's in my local language).

I have successfully reproduced the bug in two different computers, and in both the problem is gone whenever I disabled the custom scaling. Interestingly enough, Windows has a predefined set of scales, and at least for some of those I was able to successfully pass the tests.

Also just for completeness of what @Sky020 asked, I tried disabling and re-enabling all extensions in Chrome, they are completely unrelated to this problem, in both computers (both have the same set of extensions installed).

Even though it's stated in my user agent above, this is Windows 10 Pro 64-bit fully updated. I also will be closing this bug report, since it's not directly related to FCC. As a side note, however, it would be very interesting to know how one, as a HTML/CSS developer might be able to work around this issue, since it is directly related to the subject of the class.

![image](https://user-images.githubusercontent.com/5495377/77374570-d150e000-6d49-11ea-8aa9-dc823e939fb2.png)
